### PR TITLE
docs: Add Fedora/RHEL Dependency instructions

### PR DESCRIPTION
**Pull request by khaled-0**
# Description

## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `refactor:`,
      `docs:`, `chore:`, `test:`, `ci:` etc).
- [x] I have read the [Contributor Guide] and followed the process outlined for submitting PRs.
- [ ] I have updated/added tests for ALL new/updated/fixed functionality.
- [ ] I have updated/added relevant documentation and added dartdoc comments with `///`, where necessary.
- [ ] I have updated/added relevant examples in [example].

## Breaking Change

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

## Related Issues

[issue database]: https://github.com/bluefireteam/audioplayers/issues
[Contributor Guide]: https://github.com/bluefireteam/audioplayers/blob/main/contributing.md#feature-requests--prs
[Conventional Commit]: https://conventionalcommits.org
[example]: https://github.com/bluefireteam/audioplayers/tree/main/packages/audioplayers/example

### CODE REVIEW

**Comment by Gustl22** [comment]
Thank you for your contribution :D

File: packages/audioplayers_linux/README.md [hunk 1]
```diff
@@ -58,4 +58,18 @@ For Arch, simply install gstreamer and its plugins via `pacman`:
...
+sudo dnf install clang cmake ninja-build pkg-config
+```
+
+[GStreamer](https://gstreamer.freedesktop.org/documentation/installing/on-linux.html?gi-language=c):
+
+```bash
+sudo dnf install gstreamer1-devel gstreamer1-plugins-base-devel
+```
+
 You can install additional plugins as needed following [the Wiki](https://wiki.archlinux.org/title/GStreamer).
```
Comment (Gustl22) [inline packages/audioplayers_linux/README.md:75]: We may can move the last line to the `archlinux` section again, as this belongs there.

**Comment by iatndiag** [comment]
////////////////////////////// Some words about Ubuntu: ///////////////////////////////////////////////////////////

If you will try to install v.6.0.1 on official Ubuntu 22.04, libgtk-3-dev will lead you to a chain of dependencies:

gir1.2-pango-1.0   1.50.6+ds-2   without  word  "ubuntu" "1.50.6+ds-2ubuntu"  (secure version, variant NOW) e.t.c.
libpango-1.0-0
libpangocairo-1.0-0
libpangoft2-1.0-0
libpangoxft-1.0-0

libgstreamer1.0-dev  and  libgstreamer-plugins-base1.0-dev will lead you to a chain:
 
libdrm2  (2.4.110-1ubuntu1)    2.4.113-2~ubuntu0.22.04.1
libdrm-intel1
libdrm-radeon1
libdrm-nouveau2
libdrm-amdgpu1

libgbm-dev:		libgbm1  22.0.1-1ubuntu2        23.0.4-0ubuntu1~22.04.1
libgstreamer1.0-dev:    libgstreamer1.0-0   (1.20.1-1)     1.20.3-0ubuntu1
			gir1.2-gstreamer-1.0  
libudev-dev:		libudev1  (249.11-0ubuntu3.7)     249.11-0ubuntu3.11

libegl-mesa0:		libglapi-mesa  (22.0.1-1ubuntu2)    23.0.4-0ubuntu1~22.04.1

SYNAPTIC can substitude all these packages

But after reboot, Ubuntu shows black screen (extra terminal Alt+F2,Alt+F3 in normal mode), privacy violation detected.
Now OS doesn't work anymore. Good thing it's a virtual machine. 
The project is assembled, but the Files/Finder/Nautilus explorer disappears. Firefox is disabled.
And the assembled project can no longer be copied or sent anywhere.

But Ubuntu version 20.04 does NOT require downgrade packages. Confidentiality is not violated. Synaptic safely automatically selects the necessary packages and everything is installed and assembled. The same errors occur, but in version 20.04 all these errors are resolved.
////////////////////////////// End Some words about Ubuntu ///////////////////////////////////////////////////////////

**Comment by iatndiag** [comment]
I accidentally built the project not as for Android/Linux, but as for Windows with a collection of audio players. As a result, I got a gigantic list in the Ubuntu sound settings:

application - volume
application - volume
application - volume
application - volume
application - volume
application - volume
application - volume
application - volume
...
ENDLESS

When the list became simply huge, the sound disappeared completely. It turns out that when the same note is repeated, one audio player is added.

I just checked, no, I built it using a platform detector. It turned out that I built it not as for Windows, but as for Android. And all this is for Linux. Now I will have to develop a third version for Linux. On MacOS, everything seemed to work fine.

**Comment by Gustl22** [comment]
Thank you for your contribution

**Pull request merged**